static/types/default-configuration.yml

```
@@ -1,24 +1,24 @@
 plugins:
   - uses:
-      - plugin: https://ubiquity-os-command-start-stop-main.ubiquity.workers.dev
+      - plugin: https://command-start-stop-main.deno.dev
         skipBotEvents: false
         with:
           maxConcurrentTasks:
             collaborator: 999999
             contributor: 999999
   - uses:
-      - plugin: https://ubiquity-os-plugin-demo-main.ubiquity.workers.dev
+      - plugin: https://command-demo-main.deno.dev
```

```suggestion
      - plugin: https://os-command-demo.ubq.fi
```

This one I couldn't find on health.ubq.fi. I suppose its because its not on @ubiquity-os-marketplace so we don't dynamically pick it up. However the router should still route to it assuming it follows the same naming convention. 

https://os-command-demo.ubq.fi/manifest.json works!

---

@0x4007 Right this one is in `ubiquity` organization. Nice!